checka11y.css

```diff
@@ -87,6 +87,15 @@ dl > *:not( dt ):not( dd )::after {
   font-family: var(--checka11y-font, cursive);
 }
 
+dl > *:not( dt ):not( dd ) {
```

Comment:
Please revert all changes in this file and ensure it is up to date with `master`. 😊